src/main/java/org/galatea/starter/service/IexService.java

```diff
@@ -45,5 +73,50 @@
     }
   }
 
+  /**
+   * Get the last traded price for each Symbol that is passed in.
+   *
+   * @param symbol the symbol to get a historical prices for.
+   * @param range the range to get historical prices for
+   * @param date the date to get historical prices for
+   * @return a list of historical prices for the Symbol that is passed in.
+   */
+  public List<IexHistoricalPrices> getHistoricalPricesForSymbol(
+      final String symbol,
+      final String range,
+      final String date) {
+    if (symbol.compareTo("") == 0) {
+      return Collections.emptyList();
```

Comment:
Rather than returning an empty list, wouldn't the client want to know that the request is malformed?